src/data/events/alumni.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import { Container, Typography, Grid } from '@material-ui/core';
+
+export const alumni = {
```

Comment:
Any reason for adding `export`? We don't need the `export` if we aren't using the object outside of this file